sentry_sdk/tracing_utils.py

```diff
@@ -395,6 +395,16 @@ def should_propagate_trace(hub, url):
     return False
 
 
+def get_running_span_or_transaction(hub):
```

Comment:
@antonpirker  we've long wanted to just expose `current_span` and `current_transaction` on the top level api, so maybe split this out into a separate PR and just do it as well.

Reply:
Ok, sounds like a useful thing. Have put it here: https://github.com/getsentry/sentry-python/pull/1954